docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+
+If you haven't set up Salesforce DX, follow these steps:
+
+1. `Install Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.228.0.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ 
+        .. important:: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your devhub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
+
+SFDX supports multiple DevHubs, so CumulusCI uses the one set as ``defaultdevhubusername`` when creating scratch orgs.
+
+Learn more about Salesforce DX at https://developer.salesforce.com/platform/dx.
+
+
+
 Connect to GitHub
 -----------------
 
-Work on an existing CumulusCI project
+In order to allow CumlusCI to work with your CumulusCI projects in GitHub, connect GitHub as a service in ``cci``.
+
+First, `create a new personal access token <https://github.com/settings/tokens/new>`_ with both "repo" and "gist" scopes specified. (Scopes appear as checkboxes when creating the personal access token in GitHub.) Copy the access token to use as the password when configuring the GitHub service.
+
+[ EXAMPLE PHOTO OF ACCESS TOKEN TO COPY??? ]
+
+Next, run the following command and provide your GitHub username and access token as the password::
+
+    $ cci service connect github
+
+Verify the GitHub service is connected by running ``cci service list``:
+
+.. image:: images/service-list.png
+
+Once you've configured the ``github`` service it will be available to **all** CumulusCI projects.
+
+Services are stored in the global CumulusCI keychain by default.
+
+
+
+Work on an Existing CumulusCI Project
 -------------------------------------
 
-Start a new CumulusCI project
------------------------------
+Follow this section if there's an existing CumulusCI project on GitHub that you'd like to work on.
+
+.. note:: CumulusCI does not support projects stored on other Git hosts such as BitBucket or GitLab at this time.
+
+There are three preliminary steps for working on an existing CumulusCI project. You need to:
+
+* `Install CumulusCI`_
+* `Install Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+* `Clone the Project's GitHub Repository <https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories>`_
+
+You can change directories into the project's root directory and begin executing ``cci`` commands. For example, ``cci project info`` displays information about the project:
```

Comment:
```suggestion
You can change directories into the project's root directory and begin executing ``cci`` commands. For example, ``cci project info`` shows information about the project.
```
Style point for avoiding "displays": https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=DOCTEAM&title=display+vs+show